### PR TITLE
Shut down `react/display-name`

### PR DESCRIPTION
**Pull request by Primajin**
On our experience, adding the `displayName` to all the components has been irrelevant for sake of testing, debugging and ease of development.

unearthing this old branch

### CODE REVIEW

File: react.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ module.exports = {
   rules: {
     'react-hooks/exhaustive-deps': 'warn',
     'react-hooks/rules-of-hooks': 'error',
+    'react/display-name': 'off',
```
Comment (tschaub) [inline react.js:17]: This is the default behavior.

You should be able to add an example case to demonstrate that this rule is not being enforced already:
```js
import React from 'react';

export default class extends React.Component {
  render() {
    return <div>no display name</div>;
  }
}
```

Comment (Primajin) [reply react.js:17]: OK will do that thanks. I just spotted this old branch and we also have the rule set on our side but maybe that is obsolete by now, will add a test tomorrow 👍

Comment (Primajin) [reply react.js:17]: So I spot a couple of warning that are thrown in our code:

Screenshot 2020-11-10 at 09 29 47

Screenshot 2020-11-10 at 09 31 28
Screenshot 2020-11-10 at 09 31 09
Screenshot 2020-11-10 at 09 31 02

Will come up with a good example.

Comment (Primajin) [reply react.js:17]: Upon researching @bartvde found out that these are false positives https://github.com/yannickcr/eslint-plugin-react/issues/2269#issuecomment-491567533 so we fixed this on our side: https://hello.planet.com/code/product/om-web/-/merge_requests/1503 making this PR obsolete 🎉

**Pull request closed without merging**